docs/report-entries.adoc

```diff
@@ -110,4 +110,32 @@ The extension will publish your report entry if the test fails.
 ON_ABORTED::
 The extension will publish your report entry if the test was aborted (for example because of an https://junit.org/junit5/docs/current/user-guide/#writing-tests-assumptions[Assumption]).
 
-Entries will always be published after the test finished.
\ No newline at end of file
+Entries will always be published after the test finished.
+
+=== Publishing test parameters
+
+You can access `@ParameterizedTest` parameter values and publish them with the `@ReportEntry` annotation.
+Add a number (starting from zero) between curly braces to reference the parameter with that index.
+Inside the report entry, the parameters can be referenced in any order and multiple times.
+For example, if you have a `@ParameterizedTest` like the following:
+
+[source, java]
+----
+@ParameterizedTest
+@MethodSource("params")
```

Comment:
For small parameter sets, `@CsvSource` is more succinct and easier to read. Can you switch to it?

Reply:
Changed to `@CsvSource`